spec/models/movie_spec.rb

```diff
@@ -3,5 +3,7 @@
 RSpec.describe Movie, type: :model do
   describe "relationships" do
     it {should belong_to :studio}
+    it { should have_many :actors }
```

Comment:
Please also include a test for the other association through the joins record like so:
`it { should have_many :movie_actors }`
`it { should have_many(:actors).through(:movie_actors) }`